tools/build.js

```diff
@@ -225,7 +226,7 @@ Manifest-Version: 1.0
 Main-Class: foam.nanos.boot.Boot
 Class-Path: ${jars}
 Implementation-Title: ${APP_NAME}
-Implementation-Version: ${TIMESTAMP_VERSION}
+Implementation-Version: ${FOAM_BIN_VERSION}
```

Comment:
Rather than add new FOAM_BIN_VERSION and requiring setenv to be called for genJS, we could instead do this.

```
function foamBinVersion() {
  return TIMESTAMP_FOAM_BIN ? `${VERSION}-${TIMESTAMP}` : VERSION
}
```

And then just call this when we need the bin version

```
Implementation-Version: ${foamBinVersion()}`
```